Makefile

```diff
@@ -155,17 +155,14 @@ e2e-azure: install-driver
 e2e-vault: install-driver
 	bats -t test/bats/vault.bats
 
-# Update CRDs in Helm Chart
-release: manifests
-	cp config/crd/bases/* charts/secrets-store-csi-driver/templates
-	cp config/crd/bases/* deploy/
-
-	# generate rbac-secretproviderclass
-	$(KUSTOMIZE) build config/rbac -o deploy/rbac-secretproviderclass.yaml
-
 # Generate manifests e.g. CRD, RBAC etc.
 manifests: controller-gen
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=secretproviderclasses-role paths="./..." output:crd:artifacts:config=config/crd/bases
+	cp config/crd/bases/* manifest_staging/charts/secrets-store-csi-driver/templates
+	cp config/crd/bases/* manifest_staging/deploy/
+
+	# generate rbac-secretproviderclass
+	$(KUSTOMIZE) build config/rbac -o manifest_staging/deploy/rbac-secretproviderclass.yaml
```

Comment:
how does this change get into `manifest_staging/chart`

Reply:
Updated!